tests/cypress/integration/check-block-pattern-exists.test.js

```diff
@@ -0,0 +1,46 @@
+const { randomName } = require('../support/functions');
+import { compare } from 'compare-versions';
+
+describe('Command: checkBlockPatternExists', () => {
+  if (compare(Cypress.env('WORDPRESS_CORE').toString(), '5.5', '>=')) {
+    before(() => {
+      cy.login();
+      cy.deactivatePlugin('classic-editor');
+      cy.visit('/wp-admin/post-new.php');
+      cy.closeWelcomeGuide();
+    });
+
+    const testPatterns = [
+      { title: 'Quote', cat: 'text', expected: true },
+      { title: randomName(), cat: 'text', expected: false },
+      { title: 'Quote', cat: randomName(), expected: false },
+    ];
+
+    testPatterns.forEach(testCase => {
+      const shouldIt = testCase.expected ? 'should' : 'should not';
+      it(`Pattern "${testCase.title}" ${shouldIt} exist in category "${testCase.cat}"`, () => {
+        // Wait for patterns to load on the post edit page.
+        cy.wait(1000);
```

Comment:
@Sidsector9 I wonder why do we need this? Tests run successfully without this wait on my local environment. Maybe there is GitHub-related issue?

Reply:
@cadic same here. Works well locally but fails on Actions. That is what fixed the issue with WP6.0